src/new-apps/components/AppDetailsPage/Header.test.tsx

```diff
@@ -0,0 +1,112 @@
+import { appDetails } from "@dashboard/new-apps/fixtures";
+import { render } from "@testing-library/react";
+import React from "react";
+
+import Header from "./Header";
+
+const mockHeaderOptions = jest.fn();
+const mockTopNav = jest.fn();
+const mockButton = jest.fn();
+
+jest.mock("@dashboard/components/AppLayout/TopNav", () => ({
```

Comment:
**question**: why do you need to mock children components?

Reply:
**answer:** because I only want to test this component, the implementation of this component, in these Header tests I do not care about testing implementation of children components, the children components can have their own tests.